frontend/dashboard/components/DataModelsRepoList/DataModelsRepoList.tsx

```diff
@@ -27,14 +25,17 @@ export const DataModelsReposList = ({ user, organizations }: DataModelsReposList
     organizations,
   });
 
-  const { data: repos, isPending: isPendingOrgRepos } = useSearchReposQuery({
+  const { data: starredRepos = [], isPending: areStarredReposPending } = useStarredReposQuery();
+  const { data: repos, isPending: areOrgReposPending } = useSearchReposQuery({
     uid: uid as number,
     keyword: '-datamodels',
     page: 0,
   });
 
+  const dataModelRepos = repos?.data?.filter((repo) => repo.name.endsWith('-datamodels'));
```

Comment:
Should not the list already be filtered when passing the keyword as a filter to the useSearchReposQuery? 🤔  If not, maybe we should use the keyword in the filter inside the mapQueryParams instead of filtering here? 😊

Reply:
An then the const mapping from the query data can be called; `dataModelRepes`

Reply:
Yes, the server does the sorting, so this was meant as a kind of front-end validation. But I see know that it is probably just a waste of compute 😅